core/arch/mangle_shared.c

```diff
@@ -1824,7 +1824,7 @@ find_syscall_num(dcontext_t *dcontext, instrlist_t *ilist, instr_t *instr)
 
```

Comment:
nit: Please follow the commit title and body formatting conventions

Reply:
I will do.

Reply:
(Please mark comments as Resolved when done.)